### DISCUSSION TITLE
btcpay-update

### DISCUSSION THREAD

**Discussion by rgaitan1982** [btcpayserver/btcpayserver#6475] [Technical Support] [answered]
Hi, I have BTCPay Server v1.13.7+83c948d12, I see that [2.0.4] has been released, but when I try to update using WUI or using btcpay-update.sh, it tries to update to the "latest" version, but in https://hub.docker.com/r/btcpayserver/btcpayserver/tags the tag "latest" does not exist... so my server does not update properly...

**Comment by pavlenex** [comment, marked as answer]
2.0 is opt in for now, to opt in follow these instructions https://blog.btcpayserver.org/btcpay-server-2-0/#-updating-to-20
